<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>418</width>
    <height>486</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Input requested File ID</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="requestedFileId">
      <property name="placeholderText">
       <string>request file 5;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="getPathButton">
      <property name="text">
       <string>Get Path</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Path of requested Files :</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="pathRequestedFile">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
